docs/index.md

```diff
@@ -1,15 +1,17 @@
-# py42, the official Code42 Python SDK
+# py42, the Code42 Python SDK
 
 [![license](https://img.shields.io/pypi/l/py42.svg)](https://pypi.org/project/py42/)
 [![versions](https://img.shields.io/pypi/pyversions/py42.svg)](https://pypi.org/project/py42/)
 
-`py42` is a Python wrapper around the Code42 REST APIs that also provides several other useful utility methods.
-It is designed to be used for developing your own tools for working with Code42 data while avoiding the overhead
+`py42` is a Python wrapper around the Code42 REST APIs that also provides several utility methods. Use `py42` to
+develop your own tools for working with Code42 data while avoiding the overhead
 of session / authentication management.
 
 ## Features
 
-It's just generally pretty awesome
+* Managing users, orgs, and devices.
```

Comment:
Suggested update: change "orgs" to "organizations"

Reply:
Done!